packages/synapse-react-client/src/components/EvaluationFinder/EvaluationFinder.tsx

```diff
@@ -23,24 +24,35 @@ export type EvaluationFinderProps = Pick<
 export default function EvaluationFinder(props: EvaluationFinderProps) {
   const { accessType, activeOnly, selectedIds = [], onChange } = props
   const [currentPage, setCurrentPage] = useState(0)
-  const { data, isLoading, hasNextPage, fetchNextPage, isFetchingNextPage } =
-    useGetEvaluationsInfinite(
-      {
-        accessType,
-        activeOnly,
-      },
-      { keepPreviousData: true },
-    )
+  const {
+    data,
+    isLoading,
+    hasNextPage: hasNextPageOnServer,
+    fetchNextPage,
+    isFetchingNextPage,
+  } = useGetEvaluationsInfinite(
+    {
+      accessType,
+      activeOnly,
+    },
+    { keepPreviousData: true, useErrorBoundary: true },
+  )
 
   if (isLoading) {
     return <LinearProgress />
   }
   if (!data?.pages) {
-    return <></>
+    // This should never happen since errors are propagated to the error boundary
+    return (
+      <Alert severity="error">
+        An unexpected error occurred and evaluations could not be loaded
+      </Alert>
+    )
   }
```

Comment:
If `isLoading` is true and `data?.pages` is falsy, then it must be true that either 1) the query is disabled, which should not happen here or 2) an error occurred. But if an error occurs, we're relying on the error boundary for UI. So this UI is now just fallback for a case that shouldn't happen